pkg/image/utilities/utilities.go

```diff
@@ -80,25 +80,25 @@ func GatherTargetImagesAndSources(chartsPath string, imagesFromArgs []string) (I
 		return ImageTargetsAndSources{}, fmt.Errorf("could not load KDM data: %w", err)
 	}
 
-	k8sVersion1_21_0 := &semver.Version{
+	mink8sVersion := &semver.Version{
 		Major: 1,
-		Minor: 21,
+		Minor: 31,
```

Comment:
For 2.13 (which `main` branch is for) we're adding 1.34 (https://github.com/rancher/rancher/issues/51252) and dropping 1.31 (https://github.com/rancher/rancher/issues/51253) so may as well set it to `32` here.

If backporting to `release/v2.12` we will want to keep it at `31` though.

Reply:
Nice spot, thanks for that. I will add this as an additional commit and back-port without it.